packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -758,6 +758,7 @@ export class MgtPerson extends MgtTemplatedComponent {
         switch (activity) {
           case 'Busy':
           case 'InACall':
+          case 'InAConferenceCall':
```

Comment:
While this provides a fix for the immediate issue, could we also be a little proactive here and add defaults for each of the activity switch statements to set reasonable defaults when new activities are introduced?